{
  try {
    _persistence.countByMeetupsEntryId(RandomTestUtil.nextLong());
    _persistence.countByMeetupsEntryId(0L);
  }
 catch (  Exception e) {
    Assert.fail(e.getMessage());
  }
}
